.travis.yml

```diff
@@ -1,9 +1,8 @@
 sudo: false
 language: node_js
 node_js:
-  - "iojs"
-  - "0.10"
-  - "0.12"
```

Comment:
I think ubuntu 14.04's default node.js is 0.10.x. That version is still probably on many server type systems.. we could leave it in travis.yml if it doesn't cause any problems.